core/src/main/java/com/netflix/conductor/core/execution/tasks/Inline.java

```diff
@@ -78,7 +84,18 @@ public boolean execute(
             checkEvaluatorType(evaluatorType);
             checkExpression(expression);
             Evaluator evaluator = evaluators.get(evaluatorType);
-            Object evalResult = evaluator.evaluate(expression, taskInput);
+            ExecutorService executor = Executors.newSingleThreadExecutor();
```

Comment:
can you use cached threadpool here?

Reply:
Yeah. Will use fixedThreadPool because cachedThread pool has no direct way to limit the number of threads.

Reply:
ok